k9mail-library/src/main/java/com/fsck/k9/mail/internet/BinaryTempFileBody.java

```diff
@@ -125,6 +129,7 @@ public void close() throws IOException {
             try {
                 super.close();
             } finally {
+                Log.d(K9MailLib.LOG_TAG, "deleting temp file");
                 mFile.delete();
```

Comment:
This can return false, right? We should probably be checking the return value of that.